autoload/exconfig.vim

```diff
@@ -106,7 +106,7 @@ function exconfig#apply()
         let &tagrelative=0 " set notagrelative
 
         let s:old_tags=&tags
-        let &tags=fnameescape(g:exvim_folder.'/tags')
+        set tags+=fnameescape(g:exvim_folder.'/tags')
```

Comment:
change the line 109  to this may solve that problem:
let &tags=fnameescape(s:old_tags.','.g:exvim_folder.'/tags').